website/docs/language/resources/ephemeral.mdx

```diff
@@ -0,0 +1,137 @@
+
+This topic provides reference information for the `moved` block. 
+
+## Introduction
+
+_Ephemeral Resources_ are Terraform resources that are temporary, not stored in
+state, and have a unique lifecycle. Each ephemeral block describes one or more
+ephemeral resources, such as a temporary password or connection to another
+system.
+
+<!-- After page exists, add: For information about how Terraform manages ephemeral resources, refer to
+[Epehemeral Values](/terraform/language/epehemeral-values). -->
+
+## Ephemeral Resource Syntax
+
+An `ephemeral` block declares an ephemeral resource of a specific type with a
+specific local name, much like a `resource` block. Terraform uses the name when
+referring to the ephemeral resource in the same module, but it has no meaning
+outside that module's scope.
+
+### Ephemeral Resource Arguments
+
+Most of the arguments within the body of an `ephemeral` block are specific to
+the selected ephemeralresource type. The ephemeral resource type's documentation
+lists which arguments are available and how their values should be formatted.
+
+[Meta-arguments](#meta-arguments) are defined by Terraform
+and apply across all ephemeral resource types. 
+
+### Documentation for Resource Types
+
+As with resources and data sources, each provider in the [Terraform
+Registry](https://registry.terraform.io/browse/providers) includes documentation
+for ephemeral resources it supports, if any.
+
+## Meta-Arguments
+
+The Terraform language defines the following meta-arguments, which can be used
+with any ephemeral resource type to change the behavior of resources. These
+meta-arguments work the same way for resources, data sources, and ephemeral
+resources:
+
+- [`depends_on`, for specifying hidden dependencies](/terraform/language/meta-arguments/depends_on)
+- [`count`, for creating multiple resource instances according to a count](/terraform/language/meta-arguments/count)
+- [`for_each`, to create multiple instances according to a map, or set of strings](/terraform/language/meta-arguments/for_each)
+- [`provider`, for selecting a non-default provider configuration](/terraform/language/meta-arguments/resource-provider)
+- [`lifecycle`, for lifecycle customizations](/terraform/language/meta-arguments/lifecycle)
+
+Ephemeral resources to not support the `provisioner` meta-argument.
+
+## Ephemeral resource lifecycle
+
+The lifecycle of ephemeral resources is different resources and data sources.
+When provisioning ephemeral resources, Terraform will open the resource,
+optionally renew it one or more times, and then close it.
+
+1. When Terraform needs to access the ephemeral resource result, it will open
+the ephemeral resource. For example, opening a vault secret ephemeral resource
+means the Vault provider will obtain the lease and return the secret.
+
+1. If Terraform needs access to the ephemeral resource for longer than the
+expiration time enforced by the remote system, Terraform will ask the provider
```

Comment:
What is the "expiration time enforced by the remote system"? Is this a standard time? Or something dictated by the provider?
```suggestion
remote system's enforced expiration time, Terraform asks the provider
```